packages/libs/eda/src/lib/core/components/visualizations/implementations/MapVisualization.tsx

```diff
@@ -274,7 +265,15 @@ function MapViz(props: VisualizationProps<Options>) {
         showMouseToolbar={true}
         mouseMode={vizConfig.mouseMode ?? createDefaultConfig().mouseMode}
         onMouseModeChange={onMouseModeChange}
-        //DKDK: temporarily set this for testing custom zoom control at MapViz
+        // pass defaultViewport
+        defaultViewport={{
+          center: [
+            createDefaultConfig().mapCenterAndZoom.latitude,
+            createDefaultConfig().mapCenterAndZoom.longitude,
+          ],
+          zoom: createDefaultConfig().mapCenterAndZoom.zoomLevel,
```

Comment:
Hmm... I think we can do one better. I don't see a reason have a `createDefaultConfig()` function at all. It produces the same output every time, so we could just have the following, instead of https://github.com/VEuPathDB/web-monorepo/blob/b62b7049b5f71db6b61ec71a83ad322cd7b23c0e/packages/libs/eda/src/lib/core/components/visualizations/implementations/MapVisualization.tsx#L53-L64:

```typescript
const defaultConfig = {
  mapCenterAndZoom: {
    latitude: 0,
    longitude: 0,
    zoomLevel: 2,
  },
  baseLayer: 'Street',
  dependentAxisLogScale: false,
  mouseMode: 'default',
} as const;
```

Reply:
Ah, thank you for reminding me of that. I just saw that it wasn't exported where it was defined, but forgot to check the rest of the file. The `useMemo` approach would be great!